cmd/connect.go

```diff
@@ -67,18 +67,18 @@ func (o *ConnectOptions) Complete(f Factory, args []string) (err error) {
 				Anonymous: true,
 			}
 			if err = so.Complete(f, args); err != nil {
-				return err
+				return
 			}
 			if err = so.DoSetup(f); err != nil {
-				return err
+				return
 			}
 		} else if repoErr != nil {
-			return err
+			return
 		}
 	}
 
 	if err = f.Init(); err != nil {
-		return err
+		return
 	}
 
```

Comment:
unifying the way we do these returns in each `cmd.Complete`, since I was there while refactoring `DatasetMethods` and `Instance`